testing/internal/e2e/boundary/boundary.go

```diff
@@ -0,0 +1,67 @@
+// Package boundary provides methods for commonly used boundary actions that are used in end-to-end tests.
+package boundary
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/authmethods"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/kelseyhightower/envconfig"
+)
+
+type config struct {
+	Address            string `envconfig:"BOUNDARY_ADDR" default:"http://127.0.0.1:9200"`
+	AuthMethodId       string `envconfig:"E2E_PASSWORD_AUTH_METHOD_ID" default:"ampw_1234567890"`
```

Comment:
I agree it would be nice to reduce the number of environment variables that need to be set. For now, though, I would like to be explicit and use the auth method id that enos gives. We can maybe revisit this at some point in the future.